main.go

```diff
@@ -85,7 +86,7 @@ func main() {
 		if err != nil {
 			fmt.Println("Error querying database")
 			return
-		} 
```

Comment:
for `programTyep == "add"`, we should probably just return the program if we are a staker, just for safety so someone doesnt mess something up